### ISSUE TITLE
`createComment` mutation doesn't set correctly the author

### ISSUE THREAD

**Issue by aresrioja10** [wp-graphql/wp-graphql#2131]
**Describe the bug**
I'm trying to use the writeReview from GraphQL IDE and I've detected that it doesn't set correctly the author. It always set the logged user as the author.

**To Reproduce**
1. Use this example (changing the commentOn field if neccesary):
```
mutation writeReview {
  writeReview(input: {rating: 5, author: "John Doe", authorEmail: "john.doe@email.com", commentOn: 3468, content: "Lorem ipsum", date: "01/03/2021"}) {
    clientMutationId
    rating
    review {
      author {
        node {
          name
          email
        }
      }
    }
  }
}
```

2. Although we've set that the author is John Doe, the return data says that is admin@email.com (the logged user):

```
"writeReview": {
      "clientMutationId": null,
      "rating": 5,
      "review": {
        "author": {
          "node": {
            "name": "developer",
            "email": "admin@email.com"
          }
        }
      }
    }
```

**Expected behavior**
It should set the user defined in the mutation params.

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome

**Plugin Versions**
- **WooGraphQL Version:** 0.8.1
- **WPGraphQL Version:** 1.3.3
- **WordPress Version:** 5.7
- **WooCommerce Version:** 5.1.0

**Comment by justlevine** [comment]
I've reproduced this in WPGraphQL v1.7.2

```graphql
mutation {
  createComment(
    input: {
      author: "John Doe",
      authorEmail: "john.doe@email.com",
      commentOn: 56,
      content: "Lorem ipsum",
      date: "01/03/2021"
    }
  ) {
    comment {
      author {
        node {
          email
          name
        }
      }
    }
  }
}
```
![image](https://user-images.githubusercontent.com/29322304/161253751-1224c5f5-eb25-495d-bda9-cdc519679864.png)

**Comment by jasonbahl** [comment]
Closed by https://github.com/wp-graphql/wp-graphql/pull/2323

Released in v1.8.0: https://github.com/wp-graphql/wp-graphql/releases/tag/v1.8.0
